java_tools/bin2header/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+    id 'java-library'
+}
+
+repositories {
+    mavenCentral()
```

Comment:
i wonder if we really need mavenCentral from each file, would not having it in one place be enough?